htdocs/main.php

```diff
@@ -213,8 +213,14 @@ function tplFromRequest($param)
         $errorPage->assign($tpl_data);
         $tpl_data['workspace'] = $errorPage->fetch('403.tpl');
         break;
+    default:
+        header("HTTP/1.1 500 ");
```

Comment:
When I cause a 500 error, I still get to this. is it normal?
![screenshot from 2018-01-24 12-15-52](https://user-images.githubusercontent.com/15268287/35346480-5c971778-0100-11e8-877e-bbc3a6b0c858.png)

Reply:
Maybe, are you sure you are going in the default case of that switch statement?

Reply:
shouldnt any 500 get there?

Reply:
Not if it's a 500 error that's caused by something other than an exception being thrown (ie. a syntax error, or something else that's fatal, or something that gets called outside of this try block..